protx-client/src/components/ProTx/components/charts/AnalyticsStateDistribution.jsx

```diff
@@ -56,7 +52,21 @@ function AnalyticsStateDistribution({geography, selectedGeographicFeature}) {
     );
   }
 
+  const plotLabel = 'Figure 1.';
+  let plotCaptionText = [`Distribution of projected number of cases across counties in Texas. 
+  Black vertical lines indicate thresholds used to define high, 
+  medium and low risk regions for heat map on the left.`]
 
+  if (selectedGeographicFeature) {
+    if (analytics.data && analytics.data.pred_per_100k) {
+        plotCaptionText.push([`The red vertical line indicates 
+        where `, <span className='annotation-text-bold'>{countyName} County</span>, ` falls on this distribution.`]);
+    } else {
+      plotCaptionText.push([
+        <span className='annotation-text-bold'> Note: There is no data for {countyName} County.</span>,
```

Comment:
I first was thinking of suggesting "prediction" instead of "data", like "no prediction for Travis County" but we say "no data" in the table so being consistent sounds good. 👍